.golangci.yml

```diff
@@ -68,6 +67,9 @@ issues:
   max-same-issues: 50
 
 linters-settings:
+  gosec:
+    excludes:
+      - G115 # integer overflow conversion
```

Comment:
Is this a new check or what? Does it report something on our codebase?

Reply:
This is a new rule that flags integer conversion overflow. We've got a lot of conversions that trigger it, but these are always false positives. I'm disabling it so we don't have to use nolint every time we convert between int and uint because:

- Our codebase never deals with values large enough to cause an int overflow.
- We're diligent enough in our reviews to catch incorrect conversions if they do happen.

Overall, this rule is more of a annoyance than a help.

Reply:
Yeah, we also got a similar one https://github.com/go-critic/go-critic/pull/1431

Thanks!